test/verify/check-system-services

```diff
@@ -203,6 +203,22 @@ Description=Test OnBoot Timer
 [Timer]
 OnBootSec=200min
 Unit=test.service
+""")
+        self.write_file("/usr/lib/systemd/system/cockpit-system.timer",
```

Comment:
How about /run or /usr/local/? Scribbling into /usr is nasty, and won't work on CoreOS with read-only /usr.

Reply:
/usr/local sounds good, I found that `write_file` does not handle non-existant directories but `restore_file` does! So I've expanded `write_file` with an option to mkdir -p after calling `restore_file` and everything is cleaned up nicely.